source/MRViewer/MRSceneCache.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+#include "MRMesh/MRMeshFwd.h"
+#include <optional>
+
+namespace MR
+{
+
+// class to cached scene objects data
+class SceneCache
+{
+public:
+    // invalidate all cached data
+    static void invalidateAll();
```

Comment:
comment that it is called in the beginning of each frame

Reply:
fixed